src/teams/useTeams.ts

```diff
@@ -1,34 +1,41 @@
-'use client'
+"use client";
```

Comment:
Let's replace all of the double quotes with single quotes and remove semi colons.

The style plugin is now set up on the repo, so this will be the last time we have to deal with these inconsistencies xD